assemblies/assembly-add-custom-app-file-openshift.adoc

```diff
@@ -5,8 +5,8 @@ To access the {product}, you must add a custom application configuration file to
 
 [source,yaml,subs="attributes+"]
 ----
-kind: ConfigMap
-apiVersion: v1
+apiVersion: rhdh.redhat.com/v1alpha1
+kind: Backstage
```

Comment:
I don't think this is the YAML that needs to be changed. This is a valid ConfigMap manifest and seems to be common to the procedures for both Helm and Operator. The error was specific to the Operator.

![image](https://github.com/user-attachments/assets/165d12bd-ae91-4df0-9c0a-e0a559a13329)